### ISSUE TITLE
Shared Addressbook?

### ISSUE THREAD

**Issue by GinSoakedBoy** [Kozea/Radicale#1058]
Is it possible to create an Address Book that can be accessed by two accounts, one with write only access and one with read/write?

I currently have an 'admin’ user with an address book and the give a read only user access to the address book using symlink to the folder in Windows.

**Comment by wsirhc** [comment]
Same question to me, see also
[#947](https://github.com/Kozea/Radicale/issues/947#issuecomment-643684676)

**Comment by hoschi-it** [comment]
> Is it possible to create an Address Book that can be accessed by two accounts, one with write-only access and one with read & write?

Yes, according to the documentation ([radicale.org](https://radicale.org/)), that's absolutely in the scope of the this project.


> 
> I currently have an 'admin’ user with an address book and the give a read-only user access to the address book using a symlink to the folder in Windows.
> 
> The Storage paths are:
> C:\Radicale\Storage\collection-root\admin\4f0e7bd8-19b3-36c0-6b59-f396e75c1ed7

Copy the corresponding web-link (which should be `yourDomain:yourPort/admin/4f0e7bd8-19b3-36c0-6b59-f396e75c1ed7/` with yourDomain and yourPort substituted by the real ones) and manually subscribe to it. 
The only user for which the collection can be detected automatically is the user that owns it, which is admin in this case.

> 
> This is how I have my rights:
> 
> ```
> [admin]
> user: admin
> collection: .*
> permissions: RrWw
> 
> [users]
> user: users
> collection: .*
> permissions: Rr
> ```

With that setting, you are bound to the usernames "admin" and "users".
Make sure to log in with the username "users" for the readonly-user, otherwise, it won't work.

**Comment by scerazy** [comment]
Cannot get the rights to work at all, getting either 403 or 500 error

`[admin]
user: admin
collection: .*
permission: RrWw

[readonly_contacts]
user: ios
collection: ios(/.*)?
permission: Rr`

**Comment by asdf8dfafjk** [comment]
Guys, what even is a rights file? I see the syntax in documentation but not the exact filename. What is it called?
